### PR TITLE
[QNEBE-613] Added fetch command to reconnect the app to the developer

### PR DESCRIPTION
**Pull request by QFer**
* Added command fetch to get the highest latest version of the remote app files, author only.
* Changed slightly clone command to get the latest version of the remote app files
* Added/changed documentation for these commands
* Changed .pylintrc because of warnings unknown stuff
* Some changes because of pylint: validators and timeout argument for request
* Added unit tests for these commands
* Added unit tests for other methods to increase coverage somewhat

### CODE REVIEW

**Comment by hjir** [comment]
@QFer I tried the `fetch` command on the BQC applications (ids 98 and 103, the first one is published and the second one is not) and both give me this error: `Unhandled exception: KeyError('author')` - it's probably something with the manifest.json file?

**Comment by QFer** [comment]
@hjir I'm afraid there were more changes than username->email in the same commit. For example 'author' and 'email' fields from application were removed. QNE-ADK is not yet changed for that. I will add a ticket for that change.

File: .pylintrc [hunk 1]
```diff
@@ -117,7 +117,7 @@ max-module-lines=1500
 # separator` is used to allow tabulation in dicts, etc.: {1  : 1,\n222: 2}.
 # `trailing-comma` allows a space between comma and closing bracket: (a, ).
 # `empty-line` allows space-only lines.
-no-space-check=trailing-comma,dict-separator
+# no-space-check=trailing-comma,dict-separator
```
[the remark this answers is not in the document]
Comment (QFer) [reply .pylintrc:120]: Yes, deleting them is better

File: src/adk/api/remote_api.py [hunk 2]
```diff
@@ -124,47 +127,109 @@ def delete_application(self, application_id: Optional[str]) -> bool:
...
+            application_data: application data for manifest.json
+
+        """
+        application, latest_app_version = self.__get_latest_application_version(application_name)
+
+        new_application_name = new_application_name.lower()
+
+        self.__get_remote_application(application, latest_app_version, new_application_path, application_data)
+        application_data["application"]["name"] = new_application_name
+        application_data["application"]["author"] = ""
```
[the remark this answers is not in the document]
Comment (QFer) [reply src/adk/api/remote_api.py:228]: I will change them to be the same as the default manifest.

**Review by platelminto — DISMISSED** [review]

**Review by platelminto — APPROVED** [review]

**Pull request merged**